### DISCUSSION TITLE
Will Uno work on VS Code

### DISCUSSION THREAD

**Discussion by jbb26354** [unoplatform/uno#11305] [General]
Hello, 

Can Uno and its XAML Preview really work on Visual Studio Code for Mac?
I finally got the license to validate, and I can edit XAML, but the XAML preview won't show anything.

If anybody has had this problem and solved it, please advise.

Thanx,

Slowly

**Comment by jeromelaban** [comment]
Thanks for the question! What do you mean by XAML preview? Uno Platform supports XAML hot reload inside the running application, is that what you're referring to?

**Comment by jbb26354** [comment]
Yes and I've tried hot reload and it won't work. For now, I've switched to Visual Studio Community Edition and it works fine - right out of the box. No tinkering.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: Jérôme Laban ***@***.***>
Date: 2/14/23 5:33 AM (GMT-07:00)
To: unoplatform/uno ***@***.***>
Cc: John Bonifas ***@***.***>, Author ***@***.***>
Subject: Re: [unoplatform/uno] Will Uno work on VS Code (Discussion #11305)


Thanks for the question! What do you mean by XAML preview? Uno Platform supports XAML hot reload inside the running application, is that what you're referring to?

**Reply by jeromelaban** [reply]
Thanks for the update. If you can try again and let us know what you see in the "Uno Platform Hot Reload" output window in VS Code, it will be helpful.
